packages/@lwc/integration-karma/test/regression/lifecycle/index.spec.js

```diff
@@ -5,7 +5,11 @@ import NativeSloterSyntheticSlotee from 'x/nativeSloterSyntheticSlotee';
 import SyntheticSloterSyntheticSlotee from 'x/syntheticSloterSyntheticSlotee';
 import NativeSloterNativeSlotee from 'x/nativeSloterNativeSlotee';
 
-describe('W-15904769', () => {
+// Skipped currently because we reverted native custom element lifecycle as an API-versioned feature,
```

Comment:
Add a TODO comment for this?

Reply:
How about we just delete the test? We can always recover it from version control.

I kinda doubt at this point that we will try mixed-lifecycle again.